backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -169,7 +168,9 @@ def all_reviewers_approved(self) -> bool:
         )
 
     def has_short_review_process(self) -> bool:
```

Comment:
```suggestion
    def has_short_review_process(self) -> bool:
        zaaktypes_urls = (
            self.items.all()
            .select_related("zaak")
            .values_list("zaak__zaaktype", flat=True)
            .distinct()
        )

        config = ArchiveConfig.get_solo()

        return all(
            [zaaktype in config.zaaktypes_short_process for zaaktype in zaaktypes_urls]
        )
```

Performance wise this seems to have some effect (based on altered testcase, did not test with bigger data), but I mostly think it's more readable and should (?) do the same job?

Reply:
Ohh yes, nice!